### ISSUE TITLE
Unable to boot 64-bit images on Raspberry Pi 3

### ISSUE THREAD

**Issue by dderby** [maxnet/berryboot#353]
The Raspberry Pi 3 has an AArch64 CPU however BerryBoot is unable to boot images of this format.  As soon BerryBoot tries to execute the 64-bit init executable, it fails and the kernel panics.

`switch_root: can't execute 'sbin/init': Exec format error`

**Comment by maxnet** [comment]
You will need to compile your own kernel if you want aarch64 support.
By default the same kernel as Pi 2 is used.

**Comment by dderby** [comment]
Thanks, that's what I thought.  I'll try it when a get bit of free time.  I opened this issue as I thought a 64-bit compatible BerryBoot might be of interest to others who are unable to compile their own kernel and it would nice if it available for download from the BerryBoot website.

**Comment by maxnet** [comment]
Not sure if there is much demand. Think the only major distro that uses 64-bit is Suse right now.

**Comment by dderby** [comment]
True, there's not a lot of demand right now but I expect there will be as more distros are working on providing 64-bit support.  Devuan already uses 64-bit for the Pi 3, Debian is working on a 64-bit port and Fedora are intending to officially support the Pi 3 as an aarch64 device in Fedora 26.

**Comment by xilopaint** [comment]
@dderby did you get to compile the kernel? I want get [pi64](https://github.com/bamarni/pi64) to work with BerryBoot, but don't have a clue how to achieve it.

**Comment by carry0987** [comment]
Now I want to install openSUSE-Leap15.0-Arch64-ARM-Raspberry-pi,and it still give me some problem on kernel...

**Comment by symbios24** [comment]
Time for a Berryboot version that supports 64bit also
